lib/user_agent_parser/parser.rb

```diff
@@ -22,11 +24,13 @@ class Parser
 
     private_constant :FAMILY_REPLACEMENT_KEYS, :OS_REPLACEMENT_KEYS
 
-    attr_reader :patterns_path
+    attr_reader :patterns_paths
+
+    def initialize(patterns_path: nil, patterns_paths: [])
+      @patterns_paths = [patterns_path, *patterns_paths].compact
+      @patterns_paths = [UserAgentParser::DefaultPatternsPath] if @patterns_paths.empty?
```

Comment:
Should the default patterns database always be available?

```ruby
UserAgentParser::Parser.new(patterns_paths: ['/some/path/to/regexes.yaml'])
```
Something like that will exclude default patterns path. Not sure if that is a desirable behavior. More flexible one though.

Reply:
Thanks for your review and the merge of the other PR 🙏 
The behavior that I had in mind was to add a way to replace the original database completely, using a set of databases.
In our use case though we load both the original one and the custom extension. If you prefer I can change the new parameter to be something like 'extra_parrerns_paths' and remove the deprecation of 'patterns_path'. WDYT @opti ?

Reply:
@misdoro Sorry, missed that comment :(

That totally makes sense. Let's rebase it with the latest updates from `main` so it can be released.

Reply:
Ok, I've rebased against the latest changes in the master branch, pls tell me how does it look for you